nfs.go

```diff
@@ -546,14 +593,19 @@ func (n *NFSHandler) filenameLocked(h handle) (string, bool) {
 
 func (n *NFSHandler) getFileContents(ctx context.Context, h handle, readEnd uint64) ([]byte, *nfs.FileAttribute, error) {
 	n.mu.Lock()
-	ent, ok := n.readCache[h]
+	ent, ok := n.entCache.GetOk(h)
 	if ok {
-		defer n.mu.Unlock()
-		ent.timer.Reset(10 * time.Second)
-		return ent.contents, ent.attr, nil
+		blob, ok := n.blobCache.GetOk(ent.blobHash)
+		if ok {
+			n.mu.Unlock()
+			n.metricFileContentCacheHit.Add(1)
+			return blob, ent.attr, nil
+		}
```

Comment:
Question: is my understanding correct that the _two_ caches (as opposed to just one `blobCache lru.Cache[handle, []byte)` exist to ensure that the cache is immutable? If so, should we be verifying that the blob matches the blobHash at this point? Also, would there be legitimate cases where an entry can exist in `entCache` , but not in `blobCache` - should we log or something if that gets hit?